microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/CommandRetrier.java

```diff
@@ -67,7 +69,12 @@
 public class CommandRetrier {
     private static final Logger LOGGER = Logger.getLogger(CommandRetrier.class.getName());
 
-    private static final int DELAY_CORRECTION = 250;
+    private static final long   DEFAULT_DELAY_CORRECTION = 250L;
+    private static final String FT_DELAY_CORRECTION = "fault-tolerance.delayCorrection";
```

Comment:
My thoughts exactly. I went with a simple scheme because I saw security properties like `security.jersey.enabled`. I have not seen the use of the `mp.` prefix in our code.

Reply:
Then I say ship it as-is.